src/app/grants/page.tsx

```diff
@@ -33,55 +60,36 @@ export default function Grants() {
         cta={{ href: FILECOIN_URLS.grants.email, text: 'Email for more info' }}
       />
 
-      <section>
-        <Heading tag="h2" variant="xl">
-          Developer Grants
-        </Heading>
-        <p>
-          The Foundation funds talented and eager teams that want to work with
-          us to build a more robust, efficient, and decentralized web.
-        </p>
-        <ul>
-          <li>
-            <Heading tag="h3" variant="lg">
-              Open Grants
-            </Heading>
-            <p>
-              Have an idea for pushing the Filecoin ecosystem forward? Grants
-              are available to support projects that advance the Filecoin
-              ecosystem, scale users, or directly advance Filecoin’s mission to
-              preserve humanity’s most important information.
-            </p>
-          </li>
-          <li>
-            <Heading tag="h3" variant="lg">
-              Requests for Proposal
-            </Heading>
-            <p>
-              These grants generally have clearly scoped deliverables,
-              milestones, and funding limits. While there is some flexibility in
-              RFP deliverables, we expect teams will deliver what is in scope in
-              the RFP.
-            </p>
-          </li>
-          <li>
-            <Heading tag="h3" variant="lg">
-              Microgrants
-            </Heading>
-            <p>
-              The Filecoin Foundation awards grants to early-stage projects that
-              have developed initial protocols using Filecoin.
-            </p>
-          </li>
-        </ul>
-        <p>
-          For more information on grants,{' '}
-          <TextLink href={FILECOIN_URLS.grants.email}>email us</TextLink> or see
-          our{' '}
-          <TextLink href={FILECOIN_URLS.grants.github}>Github repo</TextLink>{' '}
-          for a more in-depth introduction to the grants program.
-        </p>
-      </section>
+      <PageSection
+        kicker="Opportunities"
+        title="Grants and Funding Opportunities"
+        description="The Foundation is seeking proposals for developer and data tooling, integrations, research and protocols, storage, retrieval, and the Filecoin Virtual Machine (FVM). These grants fall under the following categories:"
+      >
+        <CardLayout>
+          {grantsAndOpportunitiesData.map((card) => {
+            const { title, description, Icon } = card
+
+            return (
+              <Card
+                key={title}
+                title={
+                  <div className="flex items-center gap-3">
+                    <Icon className="h-6 w-6"></Icon>
+                    <Heading
+                      tag="h3"
+                      variant="lg"
+                      className="line-clamp-2 text-ellipsis"
```

Comment:
`Card` doesn't accept a `className` prop so this has no effect. Also, note that there are some components that accept a `className` prop but you should only use it if absolutely necessary. In fact, thinking of removing those in the near future as well. The `Card` component does have a `textIsClamped` option - but, in this case, I don't think we want any of the text to be clamped.

Reply:
Apologies - Oversight here. Wrong component. Scratch previous comment. In this case, though, we don't want the text to be clamped. That's useful, for example, for long blog post headings. In this case, we want all this copy to be visible - since the user can't open the card anywhere else either.